test/base/binding.coffee

```diff
@@ -78,6 +80,21 @@ describe("Quickdraw.Internal.Binding", ->
         )
     )
 
+    describe("_getNodePath(domNode)", ->
```

Comment:
😍 

You might consider tests that exercise the conditional behavior in your function:
* Node doesn't have an `id` property
* Node doesn't have a `className` property

... and then maybe edge cases like:
* The base case where the node _is_ the root node; it has no parents?